src/overview.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Dart overview
+description: A short introduction to Dart
+---
+
+<img style="padding: 30px; float: right; width: 300px" src="{% asset
+logo_lockup_dart_horizontal.png @path %}" alt="Dart product logo">
+
+Dart is a client-optimized language for fast apps on any platform. Our goal is
+to offer the most productive programming language for multi-platform
+development, paired with a [flexible execution runtime
+platform](https://dart.dev/platforms) for app frameworks across Google.
+
+Languages are defined by their "technical envelope": the many choices that are
+made during development that shape their capabilities and strengths. Dart is
+designed for a technical envelope that is particularly suited to client
+development, prioritizing both development (sub-second stateful hot reload) and
+high-quality production experiences across a wide variety of compilation targets
+(web, mobile, desktop).
+
+Dart also forms the foundation of [Flutter](https://flutter.dev): it provides
+the language and runtimes that power Flutter apps, but also supports many core
+developer tasks like formatting, analyzing, and testing code.
+
+## Dart: The Language
+
+The Dart language is type safe: it uses static type checking to ensure that a
+variable’s value always matches the variable’s static type, sometimes referred
+to as sound typing. Although types are mandatory, type annotations are optional
+because of type inference. The Dart typing system is also flexible, allowing the
+use of a 'dynamic' type combined with runtime checks, which can be useful during
+experimentation or for code that needs to be especially dynamic.
+
+Dart offers [sound null safety](https://dart.dev/null-safety), meaning that
+values can’t be null unless you say they can be. With sound null safety, Dart
+can protect you from null exceptions at runtime through static code analysis.
+Unlike many other null-safe languages, when Dart determines that a variable is
+non-nullable, that variable is always non-nullable: if you inspect your running
+code in the debugger, you’ll see that non-nullability is retained at runtime
+(hence "sound" null safety).
+
+This code sample showcases a number of Dart language features: libraries, async
+calls, nullable and non-nullable types, arrow syntax, generators, streams and
```

Comment:
Comma before and (twice)

Reply:
Former is deliberate (it's one clause), fixed the latter.